build.sbt

```diff
@@ -52,8 +52,8 @@ scalaVersion := scala212
 crossScalaVersions := Seq(scala212, scala210)
 sbtVersion in pluginCrossBuild := {
   scalaBinaryVersion.value match {
-    case "2.10" => "0.13.17"
-    case "2.12" => "1.2.1"
+    case "2.10" => "0.13.18"
+    case "2.12" => "1.2.8"
```

Comment:
IIRC, you usually want to target the lowest possible version here to avoid binary compatibility issues on older sbt versions. I’ve had issues in the past because sbt 1.x releases are always backwards compatible but not forwards compatible

Reply:
I also ran into https://github.com/sbt/sbt/issues/5049 on another project, that's why I didn't bump the minor, just the patch release. The problem is that these versions are not required for building, but as mentioned in 6e70250, for [scripted](https://github.com/sbt/sbt/blob/924d8e75cfb0f77da706925e5eebbe85cda7d868/main/src/main/scala/sbt/ScriptedPlugin.scala#L61) to build 2.13 (I tried to use 2.11 there but ran into another issue).

AFAIK only the 0.13 bump was necessary, so I'll try to keep 1.2.1 to limit the risks.

Reply:
> AFAIK only the 0.13.18 bump was necessary, so I'll keep 1.2.1 to limit the risks.

Actually, 1.2.7 is required to build 1.13 sources (as it's the first version that brings a Zinc with a compiler-bridge published for 2.13).

Reply:
Nevermind, I figured out I could just override `scriptedSbt` to leave `sbtVersion` untouched!